vignettes/how-to/use-addins.Rmd

```diff
@@ -0,0 +1,52 @@
+---
+title: "How-to: Rhino Add-ins"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How-to: Add-ins}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+To further streamline your development process, a collection of Add-ins has been designed to integrate with RStudio. These Add-ins provide convenient shortcuts and tools for common tasks.
+
+These Add-ins enhance the RStudio development workflow by providing quick access to essential tasks and enabling background execution, allowing for better transitions between coding and task management.
+
+This guide introduces each Rhino Add-in
```

Comment:
I miss a single sentence where a user can find Rhino addins. It should include a link to RStudio documentation and (optionally) an image.

Reply:
Added an image and a documentation to rstudio addins url